### DISCUSSION TITLE
Nyomtatvány küldhetősége - 24T1041

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#57] [Általános]
2024.12.31-én 9 óra körül a 24T1041-re az M2M-rendszeren keresztül "a bizonylattípus nem tölthető fel" hibaüzenet jött vissza.
A 24T1041 még 23:59-ig használható kellene legyen, ill. ekkor még nincs 25T1041.

**Comment by rrd108** [comment]
Én meg 01.03-án kaptam: "Nem sikerült az id=25T1042E paraméterrel a nyomtatványsablon beszerzése"

**Comment by TamasPill** [comment]
A fent jelzett hibák javításra kerültek.
